postinstall/upgrade_many.py

```diff
@@ -193,6 +193,39 @@ def ssh_unchecked_call(*args):
             log("Running '%s' on %s via ssh (allow failure)" % (' '.join(str(x) for x in args), ip))
             subprocess.call(ssh_args(args))
 
+        def ssh_call_with_output(*args):
+            log("Running '%s' on %s (%s) via ssh, capturing output" % (
+            ' '.join(str(x) for x in args), hostname, ip))
+            return subprocess.check_output(ssh_args(args))
+
+        def container_has_drives():
+            try:
+                resources = json.loads(ssh_call_with_output("weka", "local", "resources", "-C", container_name, "-J"))
+                for slot in resources['nodes']:
+                    if "DRIVES" in resources['nodes'][slot]['roles']:
+                        return True
+            except subprocess.CalledProcessError as ex:
+                log("Couldn't query resources of %s:%s, assuming there are no drive nodes on it, %s" %
```

Comment:
Is this safe?
on any issue we just assume that we don't have drives?

Reply:
not enough, I will make it safer...